src/dev/build/tasks/bin/scripts/kibana

```diff
@@ -14,11 +14,12 @@ while [ -h "$SCRIPT" ] ; do
 done
 
 DIR="$(dirname "${SCRIPT}")/.."
+CONFIG_DIR=${KIBANA_PATH_CONF:-"$DIR/config"}
 NODE="${DIR}/node/bin/node"
 test -x "$NODE"
 if [ ! -x "$NODE" ]; then
   echo "unable to find usable node.js executable."
   exit 1
 fi
 
-NODE_OPTIONS="--no-warnings --max-http-header-size=65536 ${NODE_OPTIONS}" NODE_ENV=production exec "${NODE}" "${DIR}/src/cli" ${@}
+NODE_OPTIONS="--no-warnings --max-http-header-size=65536 $(grep -v ^# < ${CONFIG_DIR}/node.options | xargs) $NODE_OPTIONS" NODE_ENV=production exec "${NODE}" "${DIR}/src/cli" ${@}
```

Comment:
Looks like an message is also thrown here when the file doesn't exist, which we should avoid for the reason mentioned in `kibana.bat`.

`bin/kibana: 25: bin/kibana: cannot open /tmp/config/node.options: No such file`

Reply:
Looks like we're still seeing this error when the file doesn't exist.

Reply:
@tylersmalley I think that is fixed now